content/vendor/guides/helm-chart.md

```diff
@@ -5,17 +5,15 @@ title: "Package a Helm Chart"
 weight: "1005"
 ---
 
-In this guide, we'll explore how you can leverage software bundled using Helm to integrate an off-the-shelf application into your KOTS app bundle. We'll use the Consul helm chart as an example in this case.
-
-This guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started) and set up a local git repo to manage your application.
+In this guide, we'll explore how you can leverage software bundled using Helm to integrate an off-the-shelf application into your KOTS app bundle. We'll use the Grafana helm chart as an example in this case. The guide is divided into three parts:
 
 - [Adding a Chart to your Application](#adding-a-chart-to-your-application)
 - [Releasing and Testing](#releasing-and-testing)
 - [Mapping Config Screen to Helm Values](#mapping-config-screen-to-helm-values)
 
 ### Prerequisites
 
-- You should have completed one of the [Getting Started Guides](/vendor/guides/#getting-started), this guide assumes you have a running instance of `kotsadm` to iterate against in either an existing or embedded cluster, and a local git checkout of your KOTS app manifests.
+This guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started) including having running instance of `kotsadm` to iterate against in either an existing or embedded cluster, and a local git checkout of your KOTS app manifests. We'll assume this is checked out in `~/helm-grafana`.
```

Comment:
Tried to de-dupe the two pre-req sections, plus made a note of what directory we're starting with.